<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>521</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>521</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SecureErase</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(30, 30, 30)
}

QLabel{
color: rgb(200, 200, 200);
background-color:rgba(0, 0, 0, 0)
}

QPushButton {
background-color:rgb(30, 30, 30);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 10px;
padding: 1px;
}

QPushButton:hover {
background-color:rgb(20, 115, 230);
	color: rgb(200, 200, 200);
border: 1px solid rgb(20, 115, 230);
}
QPushButton:pressed {
background-color:rgb(18, 100, 200);
color: rgb(200, 200, 200);
border: 1px solid rgb(18, 100, 200);
}
QPushButton:disabled {
background-color:rgb(20, 20, 20);
color: rgb(100, 100, 100);
border: 2px solid rgb(50, 50, 50);
border-radius: 10px;
}

QLineEdit{
background-color:rgb(30, 30, 30);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 10px;
padding: 1px;
}
QLineEdit:focus {
background-color:rgb(30, 30, 30);
color: rgb(200, 200, 200);
border: 2px solid rgb(20, 115, 230);
border-radius: 10px;
padding: 1px;
}

QTabWidget::pane {
color: rgb(200, 200, 200);
background: rgb(30, 30, 30);
border: 1px solid rgba(80, 80, 80, 255);
border-radius: 2px;
} 

QTabBar::tab {
color: rgb(200, 200, 200);
background: rgb(30, 30, 30);
border: 1px solid rgb(30, 30, 30);
border-radius: 2px;
padding: 8px;
} 

QTabBar::tab:selected { 
color: rgb(200, 200, 200);
background-color:rgb(18, 100, 200);
margin-bottom: -1px; 
}

QProgressBar {
background-color:rgb(30, 30, 30);
border: 1px solid rgb(80, 80, 80);
border-radius: 6px;
color:rgb(200, 200, 200);
text-align: center;
}
QProgressBar::chunk {
background-color:rgb(20, 115, 230);
border: 1px solid rgb(20, 115, 230);
border-radius: 1px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="main_tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>503</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_erase_files">
     <attribute name="title">
      <string>File(s)</string>
     </attribute>
     <widget class="QPushButton" name="btn_1_clear_files">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>178</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QListView" name="listView_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>481</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 6px;
</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>481</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select file(s) :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_1_delete_files">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>178</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete selected</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_1_add_files">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>178</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add file(s)</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select erase method :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn_1_0s">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);</string>
      </property>
      <property name="text">
       <string>Overwrite with 0s</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn_1_1s">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);</string>
      </property>
      <property name="text">
       <string>Overwrite with 1s</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn_1_random">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);</string>
      </property>
      <property name="text">
       <string>Overwrite with Random data</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>16</width>
        <height>111</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>260</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Passes  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>290</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed   :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_1_steps">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(40, 40, 40);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 6px;

</string>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cmb_1_speed">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>290</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(40, 40, 40);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 6px;
</string>
      </property>
      <item>
       <property name="text">
        <string>High</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Low</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btn_1_start">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>310</y>
        <width>71</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color:rgb(40, 40, 40);
color: rgb(200, 0, 10);
border: 2px solid rgb(200, 0, 10);
border-radius: 10px;
padding: 1px;
}
QPushButton:hover {
background-color:rgb(190, 0, 10);
color: rgb(200, 200, 200);
border: 1px solid rgb(190, 0, 10);
}
QPushButton:pressed {
background-color:rgb(170, 0, 10);
color: rgb(200, 200, 200);
border: 1px solid rgb(170, 0, 10);

}QPushButton:disabled {
background-color:rgb(40, 40, 40);
color: rgb(100, 0, 5);
border: 2px solid rgb(80, 0, 4);
border-radius: 10px;
}
</string>
      </property>
      <property name="text">
       <string>Erase !</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_free_space">
     <attribute name="title">
      <string>Free Space</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select drive :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>290</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed   :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn_2_1s">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);</string>
      </property>
      <property name="text">
       <string>Write with 1s</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>260</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Passes  :</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>16</width>
        <height>111</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn_2_0s">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);</string>
      </property>
      <property name="text">
       <string>Write with 0s</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings :</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_2_start">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>310</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color:rgb(40, 40, 40);
color: rgb(200, 0, 10);
border: 2px solid rgb(200, 0, 10);
border-radius: 10px;
padding: 1px;
}
QPushButton:hover {
background-color:rgb(190, 0, 10);
color: rgb(200, 200, 200);
border: 1px solid rgb(190, 0, 10);
}
QPushButton:pressed {
background-color:rgb(170, 0, 10);
color: rgb(200, 200, 200);
border: 1px solid rgb(170, 0, 10);

}QPushButton:disabled {
background-color:rgb(40, 40, 40);
color: rgb(100, 0, 5);
border: 2px solid rgb(80, 0, 4);
border-radius: 10px;
}
</string>
      </property>
      <property name="text">
       <string>Secure free space !</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select method :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmb_2_steps">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(40, 40, 40);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 6px;
</string>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cmb_2_speed">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>290</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(40, 40, 40);
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 6px;
</string>
      </property>
      <item>
       <property name="text">
        <string>High</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Low</string>
       </property>
      </item>
     </widget>
     <widget class="QRadioButton" name="rbtn_2_random">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);</string>
      </property>
      <property name="text">
       <string>Write with Random data</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="lbl_freespace">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>20</y>
        <width>441</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(30, 30, 30);
border: 0px solid rgb(80, 80, 80);
border-radius: 6px;</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhMultiLine</set>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ca3b3b;&quot;&gt;This process will generate and store temporary files on your preferred drive, continuously adding data until the drive reaches its full capacity.&lt;/span&gt;&lt;span style=&quot; color:#cacaca;&quot;&gt; Once the drive is full, the generated files will be deleted. This method ensures that any previously deleted files are securely overwritten, making it harder for them to be recovered.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QListView" name="listView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>112</y>
        <width>481</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(200, 200, 200);
border: 2px solid rgb(80, 80, 80);
border-radius: 6px;
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_2_refresh">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>↺</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="aniblock">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>470</y>
      <width>533</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
background-color:rgb(30, 30, 30);
border: 2px solid rgba(80, 80, 80, 0);
border-radius: 6px;
color:rgb(200, 200, 200);
text-align: center;
}
QProgressBar::chunk {
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.903727, stop:0 rgba(20, 115, 230, 255), stop:0.215909 rgba(20, 115, 230, 255), stop:0.528409 rgba(20, 115, 230, 226), stop:1 rgba(255, 255, 255, 0));
border: 2px solid rgba(20, 115, 230, 0);
border-radius: 4px;
}</string>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_icon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>IMAGE</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_version">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>v0.1.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_about">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
